File: w03/d03/khadijah_celestine/fibonacci/lib/fibonacci.rb
def fibonacci(n)
	if(n == 0)
    return 1
  elsif(n == 1)
    return 2
  else
    return fibonacci(n - 1) + fibonacci(n - 2)
	end
end

def sum_of_even_fibonaccis(n)
	sum = 0
	for nth in [*0..n]
		fib = fibonacci(nth)
		if fib < n
			if fib.even?
				sum += fib
			end
		else 
			break
		end #end check for less than limit
	end #end for
	sum
end #end def

sum_of_even_fibonaccis(5)

# use an array to hold the fib numbers
# once any given the fib number is greater than limit stop
# .select the even numbers in the array and then
# .reduce it to the sum
def sum_of_even_fibonaccis2(n)
	fibs = []
	nth = 0
	fibs[nth] = fibonacci[nth]
end

File: w06/d05/elizabeth_abernethy/morning_ex/fibonacci.js
 fibonacci = [ 0, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89 ]
 var sum = 0;

// calculate the sum of the first 10 numbers in the Fibonacci sequence


for (var i = 0; i < fibonacci.length; i++) {


  sum += parseInt(fibonacci[i])
}

console.log(sum)
